jsonnet/lib/file_paths.libsonnet

```
@@ -0,0 +1,16 @@
+{
+  //-----------------------//
+  // FILE PATH IDENTIFIERS //
+  //-----------------------//
+
+  // file path identifiers for remote desktop applications
+  remoteDesktops: [
+    // Chrome Remote desktop
+    'Chrome Remote Desktop\\.app'
```

```suggestion
    'Chrome Remote Desktop\\.app',
```

Adding a comma here should fix the formatting acceptance check.

I'll work on getting some better error message around that in the future.

---

> I'll work on getting some better error messaging around that in the future.

While it's not necessary as the offending line is listed above, I just made a change to the format script, but you'll need to close/reopen this PR to make use of it.